packages/core/src/modules/indy/repository/AnonCredsCredentialDefinitionRepository.ts

```diff
@@ -0,0 +1,28 @@
+import type { AgentContext } from '../../../agent/context/AgentContext'
+import type { Query } from '../../../storage/StorageService'
+
+import { EventEmitter } from '../../../agent/EventEmitter'
+import { InjectionSymbols } from '../../../constants'
+import { injectable, inject } from '../../../plugins'
+import { Repository } from '../../../storage/Repository'
+import { StorageService } from '../../../storage/StorageService'
+
+import { AnonCredsCredentialDefinitionRecord } from './AnonCredsCredentialDefinitionRecord'
+
+@injectable()
+export class AnonCredsCredentialDefinitionRepository extends Repository<AnonCredsCredentialDefinitionRecord> {
+  public constructor(
+    @inject(InjectionSymbols.StorageService) storageService: StorageService<AnonCredsCredentialDefinitionRecord>,
+    eventEmitter: EventEmitter
+  ) {
+    super(AnonCredsCredentialDefinitionRecord, storageService, eventEmitter)
+  }
+
+  public async getByCredentialDefinitionId(agentConext: AgentContext, credentialDefinitionId: string) {
+    return this.getSingleByQuery(agentConext, { credentialDefinitionId })
+  }
+
+  public async findByCredentialDefinitionId(agentConext: AgentContext, credentialDefinitionId: string) {
```

Comment:
```suggestion
  public async findByCredentialDefinitionId(agentContext: AgentContext, credentialDefinitionId: string) {
```

Reply:
tip: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
thx - using this already 🤦‍♂️